contracts/FIFOMarket.sol

```diff
@@ -79,6 +80,12 @@ contract FIFOMarket is
     return nrtsInQueue;
   }
 
+  function nextRemovalForSale() public view returns (uint256) {
+    // todo if our queue ever becomes an array and not a map, we need to define behavior for empty queue
```

Comment:
it'll depend on the implementation of our queue, which is highly up in the air right now as we grapple with the implementation of efficiently managing reserved removals.  However, it won't be hard either way.  Basically we'll just say that if the queue (however it's implemented) is empty, we return 0 (a token Id that no removal will ever have and the closest thing to null in solidity).  Because this is a map and the default value for a nonexistent entry is 0, we can do this in one line, but if that changes we'll just have to be careful we don't revert.

Reply:
added the length check now anyway